data/pirate jobs.txt

```diff
@@ -2465,6 +2465,51 @@ mission "Cargo Theft (South)"
 		payment 1500000
 		dialog "The pirates of <planet> take the secret cargo from your cargo hold and pay you <payment>."
 
+mission "Capture Smuggler"
+	name "Capture rogue smuggler"
+	minor
+	description `A smuggler absconded with his illegal cargo near <system>. The unknown person he stole it from wants it back. Recover the cargo from the <npc> and return both it and the smuggler to <origin> for <payment>.`
+	to offer
+		random < 15
+		"combat rating" > 2000
+		"reputation: Pirate" > 10
+	source
+		government Pirate
+	npc board
+		conversation
+			`"After neutralizing the crew and capturing the captain, you have your men sweep the ship. Ultimately they find the stolen cargo in the captain's cabin, hidden behind a false panel. The cargo is a nondescript storage container, but upon closer inspection you see that it's code-locked, interference shielded, and vacuum sealed. You return to your ship and hide the box under your bunk."`
```

Comment:
No reason for there to be quotation marks here.

This sounds weird:
> ... you see that it's code-locked, interference shielded, and vacuum sealed. ...

> ... hide the box under your bunk.

Reply:
Rewritten.